projects/aca-playwright-shared/src/api/nodes-api.ts

```diff
@@ -369,7 +383,7 @@ export class NodesApi {
   }
 
   async createLinkToFolderName(originalFolderName: string, originalFolderParentId: string, destinationParentId?: string): Promise<NodeEntry> {
-      destinationParentId = destinationParentId ?? originalFolderParentId;
+    destinationParentId = destinationParentId ?? originalFolderParentId;
```

Comment:
can we improve this line a bit? It can be just: 
destinationParentId ??= originalFolderParentId;

then originalFolderParentId will be assigned to destinationParentId only when destinationParentId is nullish so it is shorter version of actual code.